comprehension/ReadingComp.py

```diff
@@ -1,30 +1,29 @@
 from allennlp import pretrained
 
-
 class ReadingComp(object):
     """
-    ReadingComp takes as input the original input question plus the matched knowledge base article body text and
-    uses a reading comprehension model to select an appropriate answer from within the kb article
+        ReadingComp takes as input the original input question plus the matched knowledge base article body text and
```

Comment:
Are these changes using this Python formatter: https://github.com/google/yapf. That is what we used when we released Katecheo, and I think it would be good to continue to standardize our code in that way. You might already be compliant, but just checking.

Reply:
My apologies, I wasn't aware of the specific formatter. I'll update the code base.

Reply:
I've tested all the four modules locally, cleaned up the readme files and formatted the Python files using YAPF formatter.